docs/changelog.md

```diff
@@ -1,4 +1,8 @@
 # TBD
+### Breaking Changes
+* Split `networkCtx.AddService()` method in to new methods `networkCtx.RegisterService()` and `networkCtx.StartService()`
+* Merge `containerCreationConfig` and `containerRunConfig` into a new type `containerConfig` to centralize Docker container definitions
+  * Users should update all the `networkCtx.AddService()` and first call to `networkCtx.RegisterService()` passing it the `service ID` and the `kurtosisVolumeMountpoint` and then `networkCtx.StartService()` passing it the `serviceCtx` and the `containerConfig`
```

Comment:
This is quite a complex change, so is worth more details.

1) the remediations for the `addService` split should be put under that bullet point
2) any static/dynamic file stuff they were doing now needs to be done manually in between registerService and startService
3) they'll probably need to shift some stuff from ContainerCreationConfig to the new containerConfig